consensus/cometbft/service/configs.go

```diff
@@ -121,9 +122,7 @@ func DefaultConsensusParams(consensusKeyAlgo string) *cmttypes.ConsensusParams {
 	res.Feature.PbtsEnableHeight = 1
 	res.Synchrony.Precision = precision
 	res.Synchrony.MessageDelay = messageDelay
-
-	// Activate Stable Block Time (SBT) by default.
-	res.Feature.SBTEnableHeight = 1
+	res.Feature.SBTEnableHeight = cs.SbtConsensusEnableHeight()
```

Comment:
Just avoid hardcoding SBT enable height and duly use chainSpecs to pick the right value for it.
This is likely to make a difference only on devnet tests.